jobs/build/build-sync/Jenkinsfile

```diff
@@ -192,13 +192,17 @@ node('covscan') {
         if (params.SKIP_MULTI_ARCH_PAYLOAD) {
             cmd << "--skip-multiarch-payload"
         }
+        if (params.TRIGGERED_FROM_GEN_ASSEMBLY) {
+            cmd << "--triggered-from-gen-assembly"
+            cmd << "--build-number=${params.BUILD_NUMBER}"
```

Comment:
No need to pass build number here.

Reply:
Well I need to post the build number in the PR comment right so that release artists can quickly find it. Is there another way I can get the number?

Reply:
Understood! Updated!